main/exercise/question_list_ptest_admin.inc.php

```diff
@@ -0,0 +1,317 @@
+<?php 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)